Package.swift

```diff
@@ -13,6 +13,9 @@ let package = Package(
         .library(
             name: "GoodSwiftUI",
             targets: ["GoodSwiftUI"]),
+        .library(
+            name: "GRAsyncImage",
+            targets: ["GRAsyncImage"])
```

Comment:
Pozeram teraz ze tu mas 2 rozne libky co z toho projektu xcode vypluje. Nie su tam rovnake subory v oboch?

ze ked si importujem GoodSwiftUI dostanem to iste ako GRAsyncImage, aj s ostatnymi featurami (ak by tam boli).

Ale paci sa mi taka granularita, kedy si importnem len to, co naozaj potrebujem. Za ten napad +1 👍

Reply:
Pokial si importnes len **GoodSwiftUI**, tak nemas pristup k **GRAsyncImage**. 

<img width="832" alt="Snímka obrazovky 2023-06-15 o 9 17 55" src="https://github.com/GoodRequest/GoodSwiftUI/assets/114930016/917e1067-36f6-43e1-9a20-a7420cd5d88c">